pkg/controller/pxc/controller.go

```diff
@@ -348,6 +360,17 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 			nodeSet.Spec.Template.Annotations = make(map[string]string)
 		}
 		if cr.CompareVersionWith("1.1.0") >= 0 {
+			if r.serverVersion.Platform == api.PlatformOpenshift && cr.Spec.ProxySQL.ContainerSecurityContext != nil {
+				if *cr.Spec.ProxySQL.ContainerSecurityContext.Privileged {
```

Comment:
all should be tunable via annotations, PodSecurityContext in CR
maybe needed to add an option for proxySet.Spec.Template.Spec.ServiceAccountName